public/views/Search.vue

```diff
@@ -1,110 +1,287 @@
+          </button>
+          <button
+            class="search-nav-tab"
+            @click="tab = 'models'"
+            :class="[tab === 'models' ? 'active' : '']"
+          >
+            <i class="fa fa-connectdevelop"></i> Models
+            <span class="badge badge-pill badge-danger">{{
+              nbSearchModels
+            }}</span>
+          </button>
+          <button
+            class="search-nav-tab"
+            @click="tab = 'datasets'"
+            :class="[tab === 'datasets' ? 'active' : '']"
+          >
+            <i class="fa fa-table"></i> Datasets
+            <span class="badge badge-pill badge-danger">{{
+              nbSearchDatasets
+            }}</span>
+          </button>
+          <b-dropdown variant="light" size="sm">
+            <template v-slot:button-content>
+              Sort by: <i :class="sortingIcon"></i>
+              <strong>{{ sortingDisplayName }}</strong>
+            </template>
+            <!-- <b-dropdown-item-button @click="sortRecentDesc">
+              <i class="fa fa-sort-amount-desc"></i> Recent Activity
+            </b-dropdown-item-button>
+            <b-dropdown-divider></b-dropdown-divider> -->
+            <b-dropdown-item-button @click="sortNameAsc">
+              <i class="fa fa-sort-alpha-asc"></i> Name
+            </b-dropdown-item-button>
+            <b-dropdown-item-button @click="sortNameDesc">
+              <i class="fa fa-sort-alpha-desc"></i> Name
+            </b-dropdown-item-button>
+            <b-dropdown-divider></b-dropdown-divider>
+            <b-dropdown-item-button @click="sortFeaturesAsc">
+              <i class="fa fa-sort-numeric-asc"></i> Features
+            </b-dropdown-item-button>
+            <b-dropdown-item-button @click="sortFeaturesDesc">
+              <i class="fa fa-sort-numeric-desc"></i> Features
+            </b-dropdown-item-button>
+          </b-dropdown>
           <file-uploader
             class="file-uploader"
             @uploadstart="onUploadStart"
             @uploadfinish="onUploadFinish"
           ></file-uploader>
-        </div>
+        </nav>
       </div>
-    </div>
-
-    <section class="row flex-10">
-      <b-tabs
-        align="center"
-        class="search-content"
-        content-class="search-content-tab"
-      >
-        <b-tab :title="'Models (' + nbSearchModels + ')'" active>
-          <model-search-results
-            class="search-search-results"
-            :is-pending="isPending"
-          >
-          </model-search-results>
-        </b-tab>
+    </section>
 
-        <b-tab :title="'Datasets (' + nbSearchDatasets + ')'">
-          <dataset-search-results
-            class="search-search-results"
-            :is-pending="isPending"
-          >
-          </dataset-search-results>
-        </b-tab>
-      </b-tabs>
+    <!-- Main view. -->
+    <section class="row flex-1 justify-content-center">
+      <div class="col-12 col-md-8 search-content-wrapper">
+        <div
```

Comment:
The Spinner and no result page have been moved here from the two removed component.